packages/core/src/payment/strategies/adyenv2/adyenv2.ts

```diff
@@ -833,6 +833,22 @@ export type AdyenComponentState = (
     CardState | WechatState
 );
 
+export interface AdyenV2ValidationState {
+    valid?: boolean;
```

Comment:
Can we have these types as not all optional, since this means `{}` is same as `AdyenV2ValidationState` ?

Reply:
you're right! fixing